airbyte-integrations/connectors/source-bigcommerce/source_bigcommerce/schemas/products.json

```
@@ -0,0 +1,1395 @@
+        "gift_wrapping_options_list": {
+            "type": "array",
+            "description": "A list of gift-wrapping option IDs.\n",
+            "items": {
+                "type": "integer"
+            }
+        },
+        "sort_order": {
+            "type": "integer",
+            "description": "Priority to give this product when included in product lists on category pages and in search results. Lower integers will place the product closer to the top of the results.\n",
+            "minimum": -2147483648,
+            "maximum": 2147483647
+        },
+        "condition": {
+            "type": "string",
+            "description": "The product condition. Will be shown on the product page if the `is_condition_shown` field's value is `true`. Possible values: `New`, `Used`, `Refurbished`.\n",
+            "enum": [
+                "New",
+                "Used",
+                "Refurbished"
+            ]
+        },
+        "is_condition_shown": {
+            "type": "boolean",
+            "description": "Flag used to determine whether the product condition is shown to the customer on the product page.\n"
+        },
+        "order_quantity_minimum": {
+            "type": "integer",
+            "description": "The minimum quantity an order must contain, to be eligible to purchase this product.\n",
+            "minimum": 0,
+            "maximum": 1000000000
+        },
+        "order_quantity_maximum": {
+            "type": "integer",
+            "description": "The maximum quantity an order can contain when purchasing the product.\n",
+            "minimum": 0,
+            "maximum": 1000000000
+        },
+        "page_title": {
+            "type": "string",
+            "description": "Custom title for the product page. If not defined, the product name will be used as the meta title.\n",
+            "minLength": 0,
+            "maxLength": 255
+        },
+        "meta_keywords": {
+            "type": "array",
+            "description": "Custom meta keywords for the product page. If not defined, the store's default keywords will be used.\n",
+            "items": {
+                "type": "string"
+            }
+        },
+        "meta_description": {
+            "type": "string",
+            "description": "Custom meta description for the product page. If not defined, the store's default meta description will be used.\n",
+            "minLength": 0,
+            "maxLength": 65535
+        },
+        "view_count": {
+            "type": "integer",
+            "description": "The number of times the product has been viewed.\n",
+            "minimum": 0,
+            "maximum": 1000000000
+        },
+        "preorder_release_date": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Pre-order release date. See the `availability` field for details on setting a product's availability to accept pre-orders.\n",
+            "x-nullable": true
+        },
```

```suggestion
        "preorder_release_date": {
            "type": ["string", "null"],
            "format": "date-time",
            "description": "Pre-order release date. See the `availability` field for details on setting a product's availability to accept pre-orders.\n",
        },
```
This is a JSON schema not a swagger spec :) Please declare all nullable fields as I suggest above.

---

sure

---

done